core/server/services/url/UrlService.js

```diff
@@ -65,6 +68,11 @@ class UrlService {
         this.urlGenerators.push(urlGenerator);
     }
 
+    _onThemeChangedListener() {
+        this.reset(true);
```

Comment:
I think this must be false. That's you still run into trouble when switching versions?
Just a guess :)

Reply:
🤣